storage/ndb/rest-server/rest-api-server/internal/integrationtests/feature_store/handler_test.go

```diff
@@ -0,0 +1,690 @@
+/*
+ * This file is part of the RonDB REST API Server
+ * Copyright (c) 2023 Hopsworks AB
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package feature_store
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+
+	fsmetadata "hopsworks.ai/rdrs/internal/feature_store"
+	"hopsworks.ai/rdrs/pkg/api"
+)
+
+func TestFeatureStoreMetaData(t *testing.T) {
+
+	_, err := fsmetadata.GetFeatureViewMetadata("fsdb002", "sample_2", 1)
+	if err != nil {
+		t.Fatalf("Reading FS Metadata failed %v ", err)
+	}
+}
+
+func TestMetadata_FsNotExist(t *testing.T) {
+	_, err := fsmetadata.GetFeatureViewMetadata("NA", "sample_2", 1)
+	if err == nil {
+		t.Fatalf("This should fail.")
+	}
+	if !strings.Contains(err.Error(), fsmetadata.FS_NOT_EXIST.GetReason()) {
+		t.Fatalf("This should fail with error message: %s.", fsmetadata.FS_NOT_EXIST.GetReason())
+	}
+}
+
+func TestMetadata_FvNotExist(t *testing.T) {
+	_, err := fsmetadata.GetFeatureViewMetadata("fsdb002", "NA", 1)
+	if err == nil {
+		t.Fatalf("This should fail.")
+	}
+	if !strings.Contains(err.Error(), fsmetadata.FV_NOT_EXIST.GetReason()) {
+		t.Fatalf("This should fail with error message: %s.", fsmetadata.FV_NOT_EXIST.GetReason())
+	}
+}
+
+func Test_GetFeatureVector_success(t *testing.T) {
```

Comment:
General comment you are mixing _CammelCase_ and _snake_case_ which makes the code difficult to read.

Reply:
the snake case are used because it is used to identify related test case.